src/lookup.py

```diff
@@ -351,14 +332,583 @@ def getGfx(self, node, room=None, roomfile=None, path=None):
         return None
 
 
+class EntityLookup(Lookup):
+    PICKUPS_WITH_SPECIAL_SUBTYPES = (
+        PickupVariant["COLLECTIBLE"],
+        PickupVariant["TRINKET"],
+        PickupVariant["PILL"],
+    )
+
+    class EntityConfig:
+        class Parameter:
+            def __init__(self, node):
+                self.prefix = node.get("Prefix") or ""
+                self.suffix = node.get("Suffix") or ""
+                self.bitoffset = int(node.get("BitOffset") or 0)
+                self.bits = int(node.get("BitCount") or 12)
+
+                self.bitmask = (1 << self.bits) - 1
+
+                self.basevalue = int(node.get("BaseValue") or 0)
```

Comment:
sanity check: this is a cosmetic offset for the UI for the real value

Reply:
Yes. Much prefer the `ValueOffset=""` way of putting it, lol.